examples/sin_func/plot_sin.py

```diff
@@ -14,6 +14,9 @@
 
     x \\rightarrow \\sin(x)
 
+.. seealso::
+    :ref:`sphx_glr_auto_examples_sin_func_plot_sin_black_background.py` for a
+    fancier plot
```

Comment:
Maybe here you should add a text saying why this is there, for instance

"Note that sphinx-gallery automatically creates labels for each example from it's filename. You can thus refer to it from any part of the documentation, including from other examples, as here".